src/main/java/net/coderbot/patchwork/access/ModAccessTransformer.java

```diff
@@ -5,11 +5,11 @@
 import org.objectweb.asm.MethodVisitor;
 import org.objectweb.asm.Opcodes;
 
-public class AccessTransformer extends ClassVisitor {
+public class ModAccessTransformer extends ClassVisitor {
```

Comment:
I think this rename is alright, since it is true that this AT applies to the mod jar itself.